### DISCUSSION TITLE
Date and time will save differently when auto generated vs user input

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#3147] [Help]
Hi, I am having trouble saving my date reliably.
I am using the My SQL data connector.  I have used Timestamp and Datetime as my SQL data formats. I understand the Date is framed in the ISO format with the Z  suffix as the timezone. from what I can tell is the SQL timestamp and date-time format should be able to handle this format. 

What my problem is. 
When submitting a date-time automatically using  {{now}} it will save in my SQL DB perfectly the time is correct and the date is correct. when I select a date and time using the date picker the date will be correct but the time will be pushed 7 hours. I will share a video below. I have checked all of the timezones on my server, on the docker container, in the SQL instance, all are correct I'm not sure why this error is happening.  
any help would be greatly appreciated. 
  

https://user-images.githubusercontent.com/80058238/138741614-f9a2a8bd-142e-4744-b622-d335e5091158.mp4

**Comment by mjashanks** [comment]
This will be fixed in the next release: https://github.com/Budibase/budibase/issues/1504 - coming in a few days.

**Reply by KernOil** [reply]
Thank you, Sir!

**Comment by KernOil** [comment]
After the October update, I am still having this problem. when you get a chance could you look into this?

**Reply by mjashanks** [reply]
I just tested this and it's fixed for me - do you maybe need to update your app?
![image](https://user-images.githubusercontent.com/3524181/139398254-b461b681-bbe8-4d41-9d6f-62263b851c3b.png)

I didn't actually need to update mine for it to work, but it's worth doing anyway. Also, you will probably need to re-publish your app.

**Reply by KernOil** [reply]
Thank you for looking into this. I have double-checked the version and republished the app the only thing I have seen change is that now the default value of {{now}} also reads 7 hours ahead. before that one read correctly. here is a video. I can set you up an account if you want to look at the application

https://user-images.githubusercontent.com/80058238/139696710-3ea4e052-4383-4691-aa49-050852b3a4df.mp4

.

**Reply by mjashanks** [reply]
Ah, thanks, will look into this now. Sorry about that.

**Reply by KernOil** [reply]
no worries, Thank you.

**Comment by KernOil** [comment]
Hi, @mjashanks  I hope you and the team are doing well. I wanted to reach out and see if the team was able to find a fix for this and if it will be released with the monthly rollout.
**
also some info on here #1504
